airflow/settings.py

```diff
@@ -188,6 +188,16 @@ def configure_adapters():
         pass
 
 
+def configure_action_logging():
```

Comment:
Personally I think that a callback `cli_log_callback` would be simpler here as an interface, no big deal though.

Reply:
Synced offline and updated to register function instead of class instance.